ui/src/environments/environment-type.ts

```diff
@@ -56,17 +56,24 @@ export interface EnvironmentBase {
   enableCaptcha: boolean;
   // Captcha site key registered with the domain
   captchaSiteKey: string;
-  // Enable workbench footer on the signed in pages
-  enableFooter: boolean;
```

Comment:
moved these to the env-flags section because I believe they belong there.  TODO: remove these